env_config.rb

```diff
@@ -2,7 +2,7 @@
 
 require "superconfig"
 
-EnvConfig = SuperConfig.new do
+EnvConfig = SuperConfig.new(raise_exception: !ENV.fetch("ASSETS_COMPILATION", false)) do
```

Comment:
Hm, this is the only tricky part where you cannot let EnvConfig parse it for you because you're literally just creating EnvConfig...